### DISCUSSION TITLE
Bottom navigation bar

### DISCUSSION THREAD

**Discussion by RaWolFoX** [deckerst/aves#866] [Q&A] [answered]
How to edit bottom navigation bar items?. I need to replace favorites with other item. ![Screenshot_2024-01-09-00-23-07-76_33b445981d13b98f91994c60df898715.jpg](https://github.com/deckerst/aves/assets/114433842/658621ad-c249-40b7-aa16-3440fc2cddf0)

**Comment by deckerst** [comment, marked as answer]
It's not possible at the moment.

In the meantime, you can customize the drawer. And you can also make a shortcut on your device home to any filtered collection (from the Collection page ... menu).
